client/templates/includes/parameterSweepResults.pug

```diff
@@ -52,13 +52,13 @@ div#workflow-results.card.card-body
       div
 
         h5.inline Parameter Sweep
-        button.btn.btn-outline-collapse(data-toggle="collapse", data-target="#collapse-psweep", data-hook="collapse-psweep") -
+        button.btn.btn-outline-collapse(data-toggle="collapse", data-target="#collapse-psweep", id="collapse-psweep", data-hook="collapse-psweep") -
```

Comment:
The element on line 57 already has this id value I would suggest adding '-btn' to the end of the existing id value.

No element use my suggested id value